src/main/java/uk/gov/mca/beacons/api/gateways/PersonGateway.java

```diff
@@ -4,8 +4,10 @@
 import uk.gov.mca.beacons.api.dto.CreateOwnerRequest;
 import uk.gov.mca.beacons.api.jpa.entities.Person;
 
-public interface OwnerGateway {
+public interface PersonGateway {
   Person save(CreateOwnerRequest request);
 
+  Person getById(UUID id);
+
   Person findByBeaconId(UUID beaconId);
 }
```

Comment:
Generalize Owner -> Person for reusability.

Reply:
As discussed, I would leave this as `OwnerGateway` and `EmergencyContactGateway` as they are two distinct entities.

The confusion is just around the fact that we have haven't defined those domain objects yet.  This is a hangover from using JPA entities within the application vs defining separate owner/emergency contact domain objects.

If you're going to define those as entities I would just be careful with the registrations endpoint as this binds directly to the `Person` JPA entity, but the integration tests for the endpoint have broken anything.